src/index.pug

```diff
@@ -7,6 +7,10 @@ html
       body {
         background-color: #eee !important;
       }
+    script.
```

Comment:
Webpack has a built-in plugin named `ProvidePlugin`, you can use to create global variables. and since we're just using this for electron app, we can check the environment and add it only when building production package.